map/pom.xml

```diff
@@ -62,10 +74,48 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.geotools</groupId>
```

Comment:
Are the `geotools` dependencies only needed in the `test` context?

Reply:
9921bc46eaef1f2b022379c9e24cba52db5b1735